src/index.js

```diff
@@ -233,7 +190,7 @@
 
       if (isBrowser()) {
         styles.push(match.rule.style)
-        styles.push('color:inherit;')
+        styles.push('') // Empty string resets style.
```

Comment:
Should we document `// Empty string resets style.` somewhere else? Because setting `''` happens now in multiple parts of the code.